src/main/scala/org/antipathy/scoozie/builder/WorkflowBuilder.scala

```diff
@@ -19,21 +20,26 @@ private[scoozie] object WorkflowBuilder {
   def build(config: Config): Workflow = {
 
     implicit val credentials: Option[Credentials] =
-      ConfigurationBuilder.buildCredentials(config.getConfig("workflow"))
+      ConfigurationBuilder.buildCredentials(config.getConfig(HoconConstants.workflow))
 
-    val name = config.getString("workflow.name")
-    val path = config.getString("workflow.path")
+    val name = config.getString(s"${HoconConstants.workflow}.${HoconConstants.name}")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The string literal '${ .. }.${ .. }' appears 5 times in the file.](https://app.codacy.com/app/SimonJPegg/scoozie/pullRequest?prid=3072461)